fem/fespace.cpp

```diff
@@ -93,16 +92,68 @@ FiniteElementSpace::FiniteElementSpace(const FiniteElementSpace &orig,
    Constructor(mesh, NURBSext, fec, orig.vdim, orig.ordering);
 }
 
-int FiniteElementSpace::GetOrder(int i) const
+void FiniteElementSpace::SetElementOrder(int i, int p)
 {
-   Geometry::Type GeomType = mesh->GetElementBaseGeometry(i);
-   return fec->FiniteElementForGeometry(GeomType)->GetOrder();
+   MFEM_VERIFY(sequence == mesh->GetSequence(), "space has not been Updated()");
+   MFEM_VERIFY(i >= 0 && i < GetNE(), "invalid element index");
+   MFEM_VERIFY(p >= 0 && p <= MaxVarOrder, "order ouf of range");
+   MFEM_ASSERT(!elem_order.Size() || elem_order.Size() == GetNE(),
+               "internal error");
+
+   if (elem_order.Size()) // already a variable-order space
```

Comment:
ditto.

Reply:
See above.

Reply:
Not convinced, especially with the comment...